client/my-sites/plugins/plugin-list-header/index.jsx

```diff
@@ -171,8 +171,8 @@ export default React.createClass( {
 						<Button compact primary onClick={ this.props.updateAllPlugins } >
 							{
 								this.translate(
-									'Update %(numUpdates)d Plugin',
-									'Update %(numUpdates)d Plugins',
+									'Updated %(numUpdates)d Plugin',
+									'Updated %(numUpdates)d Plugins',
```

Comment:
New string `Updated %(numUpdates)d Plugin` can be replaced with `Update %(numUpdates)d Plugin` which already has *1